frontend/src/metabase/core/components/SelectButton/SelectButton.styled.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from "styled-components";
+import { color } from "metabase/lib/colors";
+import Icon from "metabase/components/Icon";
+import { forwardRefToInnerRef } from "metabase/styled-components/utils";
+
+interface SelectButtonRootProps {
+  hasValue: boolean;
+  fullWidth: boolean;
+}
+
+export const SelectButtonRoot = forwardRefToInnerRef(styled.button<
```

Comment:
Given how this function is implemented https://github.com/metabase/metabase/blob/80d81a10c3e3d9fc8de274a32713fc5184c58241/frontend/src/metabase/styled-components/utils.tsx#L7 I think it's better to use forwardDirectly in the component https://github.com/metabase/metabase/pull/19769/files#diff-fe9018cdb9e49c2425e15402ef1cfab95eac6404192a7d9688c9a809782f1e6dR18 to reduce component nesting. WDYT?